ProcessMaker/processmaker
# Implement 'Configure Template' Action for Screen Templates

## Issue & Reproduction Steps
This PR introduces the 'Configure Template' action in the Screen Template option menu, allowing users to reconfigure a screen template after creation. 

## Solution

- Refactored `configure.blade.php` file to handle configurations for both Process Templates and Screen Templates.
- Initial implementation of the `deleteMediaImages` method for templates (observation [FOUR-14664](https://processmaker.atlassian.net/browse/FOUR-14664) has been created).
- Implemented the configure method specifically for Screen Templates.

## How to Test

1. Create a New Screen Template.
2. Navigate to the screen template listing page.
3. Select 'Configure Template' from the Option menu for the desired template.
4. Verify redirection to a new page displaying the screen template configurations.
5. Update values in the screen template configurations.
6. Click 'Save'.
7. Ensure the template is saved successfully.
8. Verify proper error messages are displayed upon failure.

**Note:**
Errors may occur when removing thumbnails. An additional [observation](https://processmaker.atlassian.net/browse/FOUR-14664) ticket has been created to address this issue.

## Related Tickets & Packages
- [FOUR-14502](https://processmaker.atlassian.net/browse/FOUR-14502)

ci:next

.

## Code Review Checklist


[FOUR-14664]: https://processmaker.atlassian.net/browse/FOUR-14664?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FOUR-14502]: https://processmaker.atlassian.net/browse/FOUR-14502?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
